node/sc/bridge_manager.go

```diff
@@ -734,6 +734,40 @@ func (bm *BridgeManager) stopAllRecoveries() {
 	bm.recoveries = make(map[common.Address]*valueTransferRecovery)
 }
 
+func (bm *BridgeManager) SetValueTransferOperatorThreshold(bridgeAddr common.Address, threshold uint8) (common.Hash, error) {
+	bi, exist := bm.GetBridgeInfo(bridgeAddr)
+
+	if !exist {
+		return common.Hash{}, errors.New("bridge does not exist")
+	}
+
+	bi.account.Lock()
+	tx, err := bi.bridge.SetOperatorThreshold(bi.account.GetTransactOpts(), voteTypeValueTransfer, threshold)
+	if err != nil {
+		bi.account.UnLock()
+		return common.Hash{}, err
+	}
+	bi.account.IncNonce()
+	bi.account.UnLock()
+
+	return tx.Hash(), nil
+}
+
+func (bm *BridgeManager) GetValueTransferOperatorThreshold(bridgeAddr common.Address) (uint8, error) {
+	bi, exist := bm.GetBridgeInfo(bridgeAddr)
+
+	if !exist {
+		return 0, errors.New("bridge does not exist")
```

Comment:
We can declare an error variable for the duplicated messages.

Reply:
reflected. :)